versioned_docs/version-3.13/whats-new.md

```diff
@@ -0,0 +1,81 @@
+---
+title: What's New in RabbitMQ 3.13
+---
+<!--
+Copyright (c) 2024 Broadcom. All Rights Reserved. The term "Broadcom" refers
+to Broadcom Inc. and/or its subsidiaries.
+
+All rights reserved. This program and the accompanying materials
+are made available under the terms of the under the Apache License,
+Version 2.0 (the "License”); you may not use this file except in compliance
+with the License. You may obtain a copy of the License at
+
+https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+import {
+  RabbitMQServerVersion,
+} from '@site/src/components/RabbitMQServer';
+
+import MQTTLogo from '@site/static/img/mqtt-logo/mqtt-ver.svg';
+import KhepriLogo from '@site/static/img/khepri-logo.svg';
+
+# What's New in RabbitMQ 3.13
+
+RabbitMQ 3.13 is a new release branch [published on 22 Feb
+2024](/blog/2024/03/11/rabbitmq-3.13.0-announcement). <strong>RabbitMQ
+<RabbitMQServerVersion/></strong> is the latest patch release for this release
+branch. See the [Release Information section](/release-information) to learn
+more about all patch releases for this release branch.
+
+## MQTT 5.0 support
+
+<MQTTLogo className="floating-logo"/>
+
+MQTT is the standard protocol for the Internet of Things (IoT).
+
+IoT remote devices can have a poor network quality when connecting to the
+broker. Therefore, MQTT is lightweight: MQTT protocol headers are small to save
+network bandwidth. Since IoT devices might often disconnect and reconnect -
+imagine a car driving through a tunnel - MQTT is also efficient: Clients are
+connected and authenticated with a handshake that is shorter compared to other
+messaging protocols.
+
```

Comment:
Could we restructure/rewrite this information slightly, I think the opening information under this heading should tell the user "Hey, RabbitMQ now supports MQTT 5.0, how great is that! I don't think we need to explain that MQTT is lightweight and the problems it solves users can pull that information from the net so how about: 

MQTT, which is the standard protocol for the Internet of Things (IoT) is supported by RabbitMQ for over a decade, and now our latest 3.13.0 release supports the latest version of MQTT, which is 5.0. The list of features that come with MQTT 5.0 to further address the issues of IoT device communication is vast. For a deeper dive into MQTT 5.0 support, its specifics with RabbitMQ, and the list of features you can avail of, go through the [MQTT Overview]( (/blog/2023/07/21/mqtt5) in our MQTT blog post.

Reply:
Indeed, that's better. I took your paragraph and modified the link at the end to say:
> go through [our MQTT 5.0 blog post]( (/blog/2023/07/21/mqtt5).

Because I think the entire blog post is useful, not just the overview section.